ext/node/polyfills/os.ts

```diff
@@ -314,10 +316,36 @@ export function uptime(): number {
 
 /** Not yet implemented */
 export function userInfo(
-  // deno-lint-ignore no-unused-vars
   options: UserInfoOptions = { encoding: "utf-8" },
 ): UserInfo {
-  notImplemented(SEE_GITHUB_ISSUE);
+  const uid = Deno.uid();
+  const gid = Deno.gid();
+
+  if (isWindows) {
+    uid = -1;
+    gid = -1;
+  }
+
+  let _homedir = homedir();
```

Comment:
This is incorrect, as per node docs:
> The value of homedir returned by os.userInfo() is provided by the operating system. This differs from the result of os.homedir(), which queries environment variables for the home directory before falling back to the operating system response.

I am unsure however how to do this, and think overall this should be good enough